# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/nr_socket_prsock.h
# Commit: f18e61be455d
# Full Hash: f18e61be455d493f10a2cc651cd254859894e02e
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-02-26 03:52:35
# Regressor Bug: 1521879
# File Overlap Count: 1
# Description:
#   Bug 1521879 - Part 1: IPC-based MediaTransport implementation r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D17273
# ==============================================================================

diff -r 8136a17fbec9 -r f18e61be455d media/mtransport/nr_socket_prsock.h
--- a/media/mtransport/nr_socket_prsock.h	Mon Feb 25 21:48:20 2019 +0000
+++ b/media/mtransport/nr_socket_prsock.h	Mon Feb 25 21:50:42 2019 +0000
@@ -69,6 +69,12 @@
 #include "mozilla/TimeStamp.h"
 #include "mozilla/ClearOnShutdown.h"
 
+// nICEr includes
+extern "C" {
+#include "transport_addr.h"
+#include "async_wait.h"
+}
+
 // Stub declaration for nICEr type
 typedef struct nr_socket_vtbl_ nr_socket_vtbl;
 typedef struct nr_socket_ nr_socket;